### DISCUSSION TITLE
Snap Couldn't find coolwsd.xml file, unable to add font

### DISCUSSION THREAD

**Discussion by PuZhiweizuishuai** [nextcloud/all-in-one#2724] [Questions] [answered]
**Describe**
I deployed Nextcloud using snap and when trying to add fonts, I couldn't locate the coolwsd.xml file in the system to modify remote_font_config. How can I solve this problem?

**To Reproduce**
1. There is a temporal coolwsd.xml file but no obvious solution how to make a permanent change to it.

2.run

```
find / -name "coolwsd.xml"
```
3.result

```
/tmp/appimage_extracted_<hex-id>/etc/coolwsd/coolwsd.xml
```

**Client details:**
 - OS:   Windows10
 - Browser Edge 1114.0.1823.41 

## Server details

**Operating system:**  Ubuntu 20.04.6 LTS

**Nextcloud version:** 25.0.6 

**Version of the richdocuments app:** 7.1.4

**Version of Collabora Online:** 22.5.1301

**Comment by szaimen** [comment, marked as answer]
Hi, please ask your question at https://help.nextcloud.com as it is unrelated to Nextcloud AIO.
